src/main/java/de/internetsicherheit/brl/bloxberg/cache/ethereum/Block.java

```diff
@@ -62,11 +62,18 @@ public BlockWithData(EthBlock.Block ethBlock) {
                 .collect(Collectors.toList());
     }
 
+    public void labelTransactions(List<BlockTransaction> labelledTransactions){
```

Comment:
Yes, I need it like this. What about naming it ```setBlockTransactionsLabel```?

Reply:
I doubt that you need this, give this list as second constructor argument or something.

Reply:
I can't give it as a second argument, since they are the transactions labeled of the block I am creating. How can I assign as a parameter a transformed argument of the object I am creating? I would go for ``` setBlockTransactionsLabel ```

Reply:
No not good, if you don't see the solution, I can provide a commit. Now that I check the class, the logic of `transformTransactions()` should already move into another class and not this one. This other class can also attach the labels then, thereby allowing for a single operation creation of the `Block` class. (it is really a bad design that a `Block` object is first created, is in kind of a dirty state and then needs to be modified again by calling `setBlockTransactionsLabel` to move it into a valid state).

Reply:
ok, let me commit the things that I solved and then you commit the rest, so that I learn and we can go on.